yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@satyasashi why is this file here ? Please remove this and squash all your commits into one.

Reply:
Oh didn't check that. Should I add that file to .gitignore? can you help me with some more tips as I am new to contribution and Intermediate Git.

Reply:
Hey yeah I will delete it by tomorrow. I took accommodation in hostel for PyCon. My hotspot has connectivity issues couldn't connect to it to work on.